### PR TITLE
support relay 1.5

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: src/getWriter.js [hunk 1]
```diff
@@ -11,8 +11,25 @@ const {
...
+  schema: GraphQLSchema,
+  documents: Map<string, Object>,
+  baseDocuments: Map<string, Object>,
+  sourceControl: any,
+  reporter: any,
+}
+
 export default function getWriter (baseDir: string) {
-  return (onlyValidate: boolean, schema: GraphQLSchema, documents: Map<string, Object>, baseDocuments: Map<string, Object>) => {
+  return (config: WriterConfig | boolean, ...args) => {
```
Comment (graingert) [inline src/getWriter.js]: this comes through as an object in 1.5

File: src/index.js [hunk 2]
```diff
@@ -87,7 +87,8 @@ class RelayCompilerWebpackPlugin {
     const errors = []
     try {
       const reporter = {
-        reportError: (area, error) => errors.push(error)
+        reportError: (area, error) => errors.push(error),
+        reportTime: () => {},
```
Comment (graingert) [inline src/index.js]: new function expected on reporter

**Comment by josh** [comment]
Nice! This looks good.

Just wondering with the graphql 0.12 upgrade, how we're planning to support the `commentDescriptions` flag change?

**Comment by graingert** [comment]
Ah I don't use getSchema I generate the schema from JS

**Comment by graingert** [comment]
@josh would changing webpack-config-github to parse the schema work for you?

I'd like to deprecate passing a string into `relay-compiler-webpack-plugin`

**Comment by josh** [comment]
> I'd like to deprecate passing a string into relay-compiler-webpack-plugin

I feel like we should keep supporting passing a schema path to the options. This mirrors how the `relay-compiler` is manually invoked via CLI flags.

**Comment by danielholmes** [comment]
This looks good @graingert , I'm just waiting for the relay-compiler 1.5 to be published on npm

**Comment by graingert** [comment]
@danielholmes it's published as rc1

**Comment by danielholmes** [comment]
Aghh im so daft, was using the wrong npm version command to look for it. Thanks @graingert

**Pull request merged**